plugins/pulp_ostree/plugins/importers/steps.py

```diff
@@ -400,11 +401,17 @@ def proxy_url(self):
         port = self.config.get(importer_constants.KEY_PROXY_PORT)
         user = self.config.get(importer_constants.KEY_PROXY_USER)
         password = self.config.get(importer_constants.KEY_PROXY_PASS)
-        if host and port:
-            url = ':'.join((host, str(port)))
+        if host:
```

Comment:
I recently had to do pretty much this exact thing in nectar: https://github.com/pulp/nectar/blob/master/nectar/downloaders/threaded.py#L149

Maybe it's worth consolidating this somewhere?